### DISCUSSION TITLE
GCode preview and Giadej (DACAI)

### DISCUSSION THREAD

**Discussion by VQHe** [mriscoc/Ender3V2S1#816] [Q&A]
Hello! 

I just installed newest firmware along with newest version of Giadej (DACAI) display compilation. Should the gcode preview work with this setup? Or do I have to use the private-folder from the "display assets" link shown below?

![image](https://user-images.githubusercontent.com/128592108/227197601-7fa5d9a8-f5c1-404e-9bf6-d99bdc3a60d6.png)

I have managed to export GCode with preview from both SuperSlicer and Cura, but seems to not make difference.

Thanks!

**Comment by Jimidean23** [comment]
You should use the privates folder. Private is for the Dacai screen.  Also make sure you do the Dacai screen update also. Mine didn’t work until I applied this update.  The private folder only changed my icons.  Sent from my iPhoneJimi Anthony BandaOn Mar 23, 2023, at 7:03 AM, VQHe ***@***.***> wrote:﻿
Hello!
I just installed newest firmware along with newest version of Giadej (DACAI) display compilation. Should the gcode preview work with this setup? Or do I have to use the private-folder from the "display assets" link shown below?

I have managed to export GCode with preview from both SuperSlicer and Cura, but seems to not make difference.
Thanks!

**Reply by cjourdan87** [reply]
So with the DACAI update, the download has a private folder and a file called firmware.zlib.  Do I just put these both on the micro sd card at the same time and update the screen?

**Reply by mriscoc** [reply]
The file firmware.zlib must be in the root of the SD card to update the firmware in DACAI screens.

**Reply by VQHe** [reply]
> The file firmware.zlib must be in the root of the SD card to update the firmware in DACAI screens.

This was it, thank you!
